test/e2e/data/capi-operator/full-providers.yaml

```diff
@@ -25,7 +25,7 @@ metadata:
 spec:
   type: infrastructure
   name: azure
-  version: v1.11.5
+  version: v1.16.0
```

Comment:
```suggestion
```
This should be enough since the CAPIProvider version is latest now?

Reply:
I guess we can totally remove the version from all CAPIProvider we have in clusterctl config

Reply:
actually let's do it here